versioned_docs/version-3.0/releasenotes/release-3.0.0.md

```diff
@@ -4,3 +4,166 @@
     "language": "en"
 }
 ---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Thanks to our devoted developers and supportive community users, the much-expected Apache Doris 3.0.0 is now available!
+
+## Compute-Storage Decoupled
+
+From version 3.0.0, Doris supports a compute-storage decoupled mode, allowing users to physically isolate business workloads and separate read and write operations through multiple computing clusters. This mode substantially reduces storage costs by utilizing mature object storage or HDFS. It mitigates operational complexities such as disk balancing and data loss due to multiple BE nodes going offline.
+
+**Future-Oriented Architecture**: The architecture is selected considering complexity, performance, and scalability:
+
+**1. Reducing Complexity**
+
+- Decoupling OLAP layer semantics from storage implementation simplifies system maintenance and reduces operational complexity.
+
+- Choosing production-verified mature storage systems.
+
+- Single source of truth for metadata, greatly simplifying system complexity, such as avoiding write publishes in integrated storage-computation modes and providing strong consistency in design.
+
+**2. Performance**
+
+Capable of achieving the performance of integrated storage-computation mode.
+
+**3. Scalability**
+
+- Data scale support is no longer limited by FE memory.
+
+- Architecture can evolve and integrate both integrated and separated storage computation.
+
+- Easily supports time travel, data sharing, and load isolation.
+
+
+
+Based on these considerations, the architectural choices made include:
+
+1. A single metadata service divided into semantic and storage layers, with the storage layer choosing production-verified and transaction-capable FoundationDB.
+
+2. Storage options include mature, low-cost object storage and HDFS.
+
+3. To compensate for performance loss due to remote storage, local data caching has been introduced at computing nodes, combining the simplicity of a Shared Disk architecture with the performance of a Share Nothing architecture.
```

Comment:
If possible, information about the performance loss (with benchmarks) would be helpful.